### DISCUSSION TITLE
has anyone tested on AlmaLinux

### DISCUSSION THREAD

**Discussion by crt1111** [Dokploy/dokploy#252] [Q&A]
i wanted to know if anyone has tested deploying dokploy to AlmaLinux and how it went

**Comment by dszp** [comment]
When I tried it returns "ERROR: Unsupported distribution 'almalinux'" today and won't continue; would be nice to have support since it's got a lot of similarities to CentOS and seems like it would translate over pretty well.

**Comment by kreangsak-dev** [comment]
Just install Docker first, then run:
```
curl -sSL https://dokploy.com/install.sh | sh
```

That's it — Dokploy is now running perfectly on AlmaLinux.

**Reply by joaopedrofrech** [reply]
Does it work on Alma Linux 10? In the documentation says it's tested up to Centos 9.

**Reply by jesussuarz** [reply]
Hi, yes it can be installed on AlmaLinux 10. Here are a few important notes based on my experience:

The Dokploy install script depends on Docker. If Docker is not installed or the script fails to install it automatically, the installation will stop.

To avoid issues, it is best to install and start Docker manually before running the Dokploy installer. This ensures the script can proceed without errors.

If you're installing Dokploy on AlmaLinux and running into issues, here's a working setup:

```bash
# Install Docker
dnf install -y dnf-utils
dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
dnf install -y docker-ce docker-ce-cli containerd.io

# Enable and start Docker
systemctl enable docker
systemctl start docker

# Set your server public IP (required if no private IP is detected)
export ADVERTISE_ADDR=YOUR_PUBLIC_IP

# Install Dokploy
curl -sSL https://dokploy.com/install.sh | sh
```

⚠️ Notes:

* Make sure Docker is **running**, not just installed
* On many VPS setups (no private IP), you **must set `ADVERTISE_ADDR` manually**
* Ports **80, 443, and 3000** must be free

After installation, access Dokploy at:
http://YOUR_PUBLIC_IP:3000
